public/scripts/main.js

```diff
@@ -130,9 +133,9 @@ const gameHandler = (function gameIIFE() {
   }
 
   function displayRole(role) {
-    document.querySelector('#image').src = role.image
-    document.querySelector('#name').textContent = role.name
-    document.querySelector('#description').textContent = role.description
+    image.src = role.image
+    name.textContent = role.name
+    description.textContent = role.description
```

Comment:
So TIL `textContent ` is better than `innerText`. Would you be able to change all uses of `innerText` to `textContent`?